res/css/style.css

```diff
@@ -220,45 +219,52 @@ body, #root, .profileViewer {
   color: transparent;
 }
 
-.treeRowToggleButton {
-  background-image: url(./treetwisty.svg);
-}
-
 .focusCallstackButton {
-  background-image: url(./circlearrow.svg);
+  background-image: url(../img/svg/circlearrow.svg);
   margin-left: 5px;
   visibility: hidden;
 }
 
-.treeRowToggleButton:active:hover,
 .focusCallstackButton:active:hover {
   background-position: -16px 0;
 }
 
-.treeRowToggleButton.collapsed {
-  background-position: 0 -16px;
+.treeViewBody:focus .treeViewBodyInner .treeViewRow.selected > .focusCallstackButton {
+  background-position: -32px 0;
 }
 
-.treeRowToggleButton.collapsed:active:hover {
-  background-position: -16px -16px;
+.treeViewBody:focus .treeViewBodyInner .treeViewRow.selected > .focusCallstackButton:active:hover {
+  background-position: -48px 0;
 }
 
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .treeRowToggleButton,
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .focusCallstackButton {
-  background-position: -32px 0;
+
+.treeRowToggleButton {
+  height: 0;
+  width: 0;
+  color: #888;
+  border-left: 8px solid;
+  border-top: 5px solid transparent;
+  border-bottom: 5px solid transparent;
+  border-right: 3px solid transparent;
+  margin-left: 5px;
+  margin-right: 5px;
 }
 
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .treeRowToggleButton.collapsed {
-  background-position: -32px -16px;
+.treeRowToggleButton.expanded {
+  transform: rotate(90deg);
+  margin-top: 5px;
 }
 
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .treeRowToggleButton:active:hover,
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .focusCallstackButton:active:hover {
-  background-position: -48px 0;
+.treeRowToggleButton:active:hover {
```

Comment:
do we really want/need `:hover` ? When using the mouse this is obviously always true, but when using the keyboard this wouldn't be triggered.
Obviously same comment in other similar places.

Reply:
If you mousedown on a button, then move your mouse outside of the button, and then let go, the button is not clicked. Consequently, during that time, it shouldn't be displayed in the pressed state. In that state, the button is `:active` but not `:hover`. So the general rule of thumb is: For buttons that do something on click, display the pressed state during `:active:hover`.

Reply:
(My argument doesn't really apply at the moment because we currently activate these buttons on mousedown instead of on click (mouseup), but I think we should fix that, so I filed #984.)

Reply:
I understand what you mean, but now I'm confused because this goes against every existing behaviors (eg: native buttons). Not that I'm against what you say.

For this patch let's keep :hover though, I don't want to bring controversial changes in this patch :)